README.md

```diff
@@ -18,7 +18,53 @@ In fact, to run tests or basically anything with Django management commands, you
 
 ### Environmental Settings
 
-A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make tests` outside of a Docker container, for example.
+A sample development environment settings file is in `.environ/.env.dev`. You can copy this over to your root as `.env` to use if you'd like to run `make test` outside of a Docker container, for example.
+
+### JIRA Worklogs Integration
+
+Invoices can be loaded with line items derived from JIRA Tempo worklogs, given that the username of the provider on the invoice is equal to a username on a JIRA instance.
+
+Syncrhonization is guaranteed -- any line item that's tagged as a JIRA worklog will be checked against incoming JIRA worklogs, and a mismatch leads to deletion, whereas ones that don't exist yet always get added, thus ensuring a complete match for all relevant fields -- issue title, worklog description, worklog quantity, and so on.
```

Comment:
1. Automatically. Or the user could do it, too.
2. Line items that aren't JIRA worklogs might not make sense to mention here since this part is specifically for JIRA worklogs integration. Tagging is meant specifically to separate JIRA worklog line items from everything else, so their concerns are separate.
3. The option is disabled by default, but it's hard to miss because it's at the bottom of the Invoice Django Admin near the save button, plus it's temporary so I'm not going to mention it.